### PR TITLE
new package: emacs-nox 26.3

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by a-schaefers** [comment]
decided against the mods i had, going basically vanilla for both Emacses

File: community/emacs-nox/build [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    --without-x \
+    --with-x-toolkit=no \
+    --with-x=no
+
+mkdir -p "$1/usr/share/emacs/site-lisp"
+cat << EOF > "$1/usr/share/emacs/site-lisp/site-start.el"
+;; fix ugly
+(menu-bar-mode -1)
+EOF
+
```
Comment (dylanaraps) [inline community/emacs-nox/build]: Missing standalone `make`.

Comment (a-schaefers) [reply community/emacs-nox/build]: fixed

File: community/emacs/build [hunk 2]
```diff
@@ -1,8 +1,6 @@
 #!/bin/sh -e
 
 ./configure \
-    --prefix=/usr \
-    --with-gnutls=no
+    --prefix=/usr
 
-make
 make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/emacs/build]: This file is from the regular `emacs` package. :+1:

Comment (a-schaefers) [reply community/emacs/build]: errg

Comment (a-schaefers) [reply community/emacs/build]: thank you for teaching me btw

**Comment by a-schaefers** [comment]
good to go here

File: community/emacs/build (hunk 2 above)
Comment (dylanaraps) [reply community/emacs/build]: No problem. :)

**Pull request merged**